Source/WarpX.H

```diff
@@ -1339,7 +1339,8 @@ private:
     int num_injected_species = -1;
     amrex::Vector<int> injected_plasma_species;
 
-    amrex::Real const_dt = amrex::Real(0.5e-11);
+    amrex::Real const_dt;
+    bool const_dt_specified;
```

Comment:
Let's use a `std::optional<amrex::Real>` as type and call the member variable `m_const_dt` according to our conventions?

Reply:
ping @dpgrote when you have the time :)